examples/features/unmanaged/3_torch_distributed.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+# To run:
+# python -m torch.distributed.run --nnodes=1 --nproc_per_node=2 \
+#  --master_addr 127.0.0.1 --master_port 29400 --max_restarts 0 \
+#  3_torch_distributed.py
+
+import logging
+import random
+
+import torch.distributed as dist
+
+import determined
+import determined.core
+from determined.experimental import core_v2
+
+
+def main():
+    dist.init_process_group("gloo")
+
+    logging.basicConfig(format=determined.LOG_FORMAT)
+    logging.getLogger("determined").setLevel(logging.INFO)
+    distributed = determined.core.DistributedContext.from_torch_distributed()
```

Comment:
nit / optional:

have you considered exposing symbols from `det.core` in `det.experimental.core_v2`?  Seems odd to make users use both.  If I were a user, I would expect `core_v2` to be a complete solution, even if some parts were copy/pasted or aliased into the namespace.

Reply:
this was working before but it got axed by the "you can only import modules" rule. I agree it is useful.

Reply:
Yeah, I think importing symbols to explicitly create a desirable user-facing namespace is good and correct.  If the linter isn't letting us build the user-facing API that we want, I think the linter needs to either get ignored in those specific cases, or removed altogether.